webapp/channels/src/components/threading/virtualized_thread_viewer/virtualized_thread_viewer.scss

```diff
@@ -0,0 +1,12 @@
+.virtual-list{
+    &__ctr{
+        display: flex;
+        height: 100%;
+        flex-direction: column;
+    }
+
+    &__content{
+        margin-bottom: 8px;
```

Comment:
@AshishDhama can you clarify what this margin is for? It seems to be adding extra space that may not be needed

![image](https://github.com/user-attachments/assets/1f57d62e-3fcd-453b-8b6f-6bb7082b832c)